tests/unit/test__instrumentation.py

```diff
@@ -63,3 +63,16 @@ def test_truncate_long_values(self):
 
         self.assertEqual(expected_name, self._get_diagonstic_value(entry, "name"))
         self.assertEqual(expected_version, self._get_diagonstic_value(entry, "version"))
+
+    def test_drop_labels(self):
```

Comment:
Would it be possible to add a test which checks that only required fields are present?

Reply:
yup, added an assert